kernel/src/syscall.rs

```diff
@@ -494,6 +511,18 @@ impl SyscallReturn {
         res.into_inner()
     }
 
+    /// Construct either SuccessU32 or SuccessU64 depending on platform.
```

Comment:
This should also come with a corresponding update to TRD 104, to promote this to the canonical Success type being used in the kernel for variable-width (word-sized) return values.

Reply:
I have removed this constructor now as SucessUsize is now an actual variant.